aruco_ros/src/marker_publish.cpp

```diff
@@ -202,9 +225,20 @@ class ArucoMarkerPublisher
               transform = static_cast<tf::Transform>(cameraToReference) * transform;
               tf::poseTFToMsg(transform, marker_i.pose.pose);
               marker_i.header.frame_id = reference_frame_;
+
+              tf::TransformBroadcaster br;
+              br.sendTransform(tf::StampedTransform(transform, ros::Time::now(), reference_frame_,
+                                                    "/marker_"+int_to_string(marker_i.id)));
+              // printf("%s %s\n", reference_frame_.c_str(), ("/marker_"+int_to_string(marker_i.id)).c_str());
             }
           }
 
+          if (markers_.size()>0)
```

Comment:
This is just for debugging. This information is being published. Please remove it.